INAD_API_extraction.yaml

```diff
@@ -93,6 +105,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/Errore'
         default:
+          # da verifica, l'attuale designe ammette risposte per per differenti http status
```

Comment:
TODO

Reply:
Per gli errori il modello json è unico (#/components/schemas/Errore) che presenta gli attributi utili per descrivere l'errore:

  "status": "<HTTP_CODE>"
  "type": "<HTTP_STATUS>"
  "detail": "<detail_error>"

E' corretto in questo modo?

Reply:
qui è in negli altrivolevo esprimere il dubbio sulla definizione del comportamento default, ad esempio la definizione 

default:
          description: RispostaRichiestaElencoDomiciliDigitali
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/RispostaRichiestaElencoDomiciliDigitali'


ad esempio indica gli http status non indicati restituiscono lo schema RispostaRichiestaElencoDomiciliDigitali... non mi trovo

Reply:
Il tag default è definito in automatico da swagger. Disabilitiamo il comportamento di default.